### DISCUSSION TITLE
New Shopify 2.0 version?

### DISCUSSION THREAD

**Discussion by chunkysteveo** [kondasoft/ks-bootshop#50] [Ideas] [answered]
Hey, hope you are well?

Have you looked into, or are you looking into making a BS theme which works with the new "OS 2.0" updates and features from Shopify? So it's all JSON and non-sectioned etc?

Loving the V4 and the V5 BS themes, and was curious if you've had any though on making another BS theme that's built from the ground up for all the new Shopify 2.0 features and functions?

Cheers, Steve

**Comment by alvinkonda** [comment]
Hi Steve,
For the moment I'm working on some other projects and frankly, I have not found the time to put into this project. But if you need any custom solutions or work for Shopify let me know. 

Regards,
Alvin

**Reply by chunkysteveo** [reply]
Thanks Alvin!

Keep up the great work!

Steve

**Comment by alvinkonda** [comment, marked as answer]
This theme has been updated and it fully supports Shopify Online Store 2.0

**Reply by chunkysteveo** [reply]
Oh hey!!! This is great news!!! Let me check out the new updates on the theme. Thanks again for your efforts on this BS theme 👌🏻😎
